<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Images_qW</class>
 <widget class="QWidget" name="Images_qW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>4Safety</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QPushButton" name="new_entry_qPB">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>Nova Entrada</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="close_qPB">
     <property name="text">
      <string>Fechar</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="view_report_qPB">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>Exibir Relatorio</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="Images_qGL">
     <item row="3" column="0">
      <widget class="QScrollArea" name="scrollArea">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>778</width>
          <height>374</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QGridLayout" name="Gallery_qGL">
           <property name="spacing">
            <number>10</number>
           </property>
           <item row="0" column="0">
            <widget class="QCalendarWidget" name="calendarWidget"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="title_qL">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Consultar Listas de Chamada</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="course_qL">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Turma</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="selected_date_qL">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Data Selecionada: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>view_report_qPB</tabstop>
  <tabstop>new_entry_qPB</tabstop>
  <tabstop>close_qPB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
